streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -62,6 +61,19 @@ class ReceivedBlockHandlerSuite
   var blockManagerMaster: BlockManagerMaster = null
   var blockManager: BlockManager = null
   var tempDirectory: File = null
+  var storageLevel = StorageLevel.MEMORY_ONLY_SER
```

Comment:
I can even set the StorageLevel back to default value in Before block which can reset the global variable before every test. Anyone write new test can see the before block to check what is global state.